src/molecules/Hero/hero.module.css

```diff
@@ -0,0 +1,16 @@
+.hero {
+  padding: 20px 28px;
+  background-color: var(--hero-background-color);
+  color: var(--hero-text-color);
+}
+
+.hero h1 {
+  margin-bottom: 28px;
```

Comment:
We need to use a spacer component. Components should not have vertical margin values greater than 0.

Reply:
Create a component Margin with a fixed size?

Reply:
We talked about that in another comment.

Reply:
```css
.verticalSpacing1x {
    margin-bottom: var(--v-spacing-1x);
}

.verticalSpacing2x {
    margin-bottom: var(--v-spacing-2x);
}
```

```jsx
export default ( { size } ) => <div className={styles[`verticalSpacing${size}`]}></div>
```